Movement Changes and Material Updates

torirossini opened the pull request:
**Better movement and updates to the materials for colors.**

Materials are now set to 0 smoothness
rigidbody uses AddForce instead of setting its velocity value!

cso7654 committed:
Add files via upload

cso7654 committed:
Add files via upload

cso7654 committed:
Add files via upload

cso7654 committed:
Restored with GitHub Desktop

cso7654 committed:
Refactored scripts

Changed public variables to [SerializeField} private variables. Added properties to said variables. Removed UI from EthanScene.  Added more comments.

cso7654 committed:
Changed HUD Layout

Shrunk health bar and color indicator. Disabled charge on crosshair and made it transparent to increase visibility.

LegendRK committed:
Created DemoScene, fixed some bugs with firing, adjusted hitbox of player and scale of overall player ship

EthanNichols committed:
Added Enemy spawning, updated enemy prefab

cso7654 committed:
Changed opacity of Health/Color indicators

from 255 to 150. Will add dynamic opacity later when needed.

cso7654 committed:
replaced health bar with slider

it's more complicated now but it still looks the same

LegendRK committed:
Changed some stuff.

Commit:
Better movement and material changes

torirossini committed:
Removing Debug line

jmortiger reviewed (approved):
It might be best to use FixedUpdate instead of Update, but that probably won't be a problem, and definitely not a hard one to fix if it ever comes up.

jmortiger commented:
![](https://github.trello.services/images/mini-trello-icon.png) [Movement Changes and Materials Update](https://trello.com/c/4STl9vXf/18-movement-changes-and-materials-update)

jmortiger commented:
![](https://github.trello.services/images/mini-trello-icon.png) [Movement Changes and Materials Update](https://trello.com/c/4STl9vXf/18-movement-changes-and-materials-update)

cso7654 committed:
Merge branch 'UI' into MVISetup

cso7654 committed:
Linked Player to UI

Health bar and color indicator now reflect color and death status of the player. Crosshair added, will rotate when hovering over enemy in later push.

cso7654 committed:
Added Menus

Scenes switch from a main menu to the main game and to a game over, then back to main menu

cso7654 committed:
Merge branch 'UI' into MVISetup

cso7654 committed:
Added Menu and Game Over

Game now starts on Main Menu if built and has buttons to go between menu states and game
